packages/scroller/src/vaadin-scroller.d.ts

```diff
@@ -14,13 +15,23 @@ import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mix
  *   <div>Content</div>
  * </vaadin-scroller>
  * ```
+ * The following attributes are exposed for styling:
+ *
+ * Attribute    | Description
+ * -------------| -----------
+ * `focus-ring` | Set when the element is focused using the keyboard.
  */
-declare class Scroller extends ThemableMixin(ElementMixin(HTMLElement)) {
+declare class Scroller extends FocusMixin(ThemableMixin(ElementMixin(HTMLElement))) {
   /**
    * This property indicates the scroll direction. Supported values are `vertical`, `horizontal`, `none`.
    * When `scrollDirection` is undefined scrollbars will be shown in both directions.
    */
   scrollDirection: 'horizontal' | 'vertical' | 'none' | undefined;
+
+  /**
+   * Indicates whether the element can be focused and where it participates in sequential keyboard navigation.
+   */
+  tabindex: number | undefined | null;
```

Comment:
The reason for allowing `null` and `undefined` in `TabindexMixin` is that we remove tabindex when disabled.
I would prefer using `number` from here to align with the native `tabIndex` property.

Not a blocker, can be also kept this way for consistency.

Reply:
Done